documentation/asciidoc/stories/assembly_batch_cr.adoc

```diff
@@ -0,0 +1,15 @@
+ifdef::context[:parent-context: {context}]
+[id='batch_operations']
+:context: batch
+= Running Batch Operations
+{ispn_operator} provides a `Batch` CR that lets you create {brandname} resources in bulk.
+Batch operations use the {brandname} command line interface (CLI) to carry out different sequences of actions to create and modify resources.
```

Comment:
It could be worth stating that it uses the CLI's batch mode and then providing a link to that documentation?

Reply:
True but there is a link to the CLI batch docs under the Batch Operations reference topic. I prefer to not link to external docs in the user story because that risks getting users off track.

I've reworded this a little though.